### DISCUSSION TITLE
Fortinet logs seemingly not reaching Logstash on manager

### DISCUSSION THREAD

**Discussion by Rbartram** [Security-Onion-Solutions/securityonion#5399] [Unsupported Versions] [answered]
Hello, 

I've enabled the fortinet module in filebeat on my sensor/forwarder server and configured my Fortigate FW to send syslogs to sensor/forwarder server via port 9004.

"tcpdump port 9004" on the sensor server shows a lot of traffic coming from the firewall, but nothing is landing in logstash/kibana.  
"tcpdump port 5644" on the sensor server reveals very little traffic being sent to logstash on the manager server.  

It did work for a while yesterday, but it stopped working when I tried to configured the azure module for filebeat.  I have since reverted my additions for azure, but it is still not working.

Filebeat and Logstash debug logs are revealing anything meaningful to me.

Filebeat config from /opt/so/saltstack/local/pillar/minions/igt-pgh-sof-01_sensor.sls"
----------

      filebeat:
        third_party_filebeat:
          modules:
            fortinet:
              firewall:
                enabled: true
                var.input: udp
                var.syslog_host: 0.0.0.0
                var.syslog_port: 9004

Elasticsearch/Logstash config from filebeat.yml on the sensor server
----------

     output.logstash:
       enabled: true
       hosts: ["manager:5644"]
       worker: 1
       bulk_max_size: 2048
       ssl.verification_mode: full
       ssl.supported_protocols: [TLSv1.0, TLSv1.1, TLSv1.2]
       ssl.certificate_authorities: ["/usr/share/filebeat/intraca.crt"]
       ssl.certificate: "/usr/share/filebeat/filebeat.crt"
       ssl.key: "/usr/share/filebeat/filebeat.key"
     setup.template.enabled: false

Any direction on were to look would be great.

Thank you, ...Rob

**Comment by weslambert** [comment, marked as answer]
To be clear, when you say it was "working", were logs flowing in, being parsed, etc? Did you actually allow access to port 9004 from an external system?

**Reply by Rbartram** [reply]
Hi.  I ended up starting a new question with based on additional understanding that I've gathered.  If you would be so kind as to look at question https://github.com/Security-Onion-Solutions/securityonion/discussions/5434

It the issue appears to have something to do with port redirection stopping from the physical interface to the docker0 interface.

**Reply by Rbartram** [reply]
Can this question be closed somehow as to eliminate the confusion that I created by creating a new question?

**Reply by dougburks** [reply]
This discussion has been marked as `Answered`.
